notebooks/introduction/categorical_distribution.ipynb

```diff
@@ -0,0 +1,656 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    theta = [1/6]*6
</pre><p>Would you like to mention just above this what is the interpretation of this theta? For example, Die is unbiased here because all probabilities are same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/code-first-ml/book1/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>